dataland-frontend/tests/e2e/specs/search-company-framework-data/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -51,12 +51,12 @@ describe("As a user, I expect the search functionality on the /companies page to
         "eq",
         getBaseUrl() +
           "/companies?" +
-          `framework=${DataTypeEnum.Gdv}` +
```

Comment:
Do you know what controls the order of these? Is it the enum? Why is the DataTypeEnum ordered like this?